### DISCUSSION TITLE
Is it possible to specified stack as a function parameter?

### DISCUSSION THREAD

**Discussion by 2A2U** [Vector35/binaryninja-api#5471] [Q&A] [answered]
Hi guys, is it possible to specified a stack as the parameter of a function like push operation before call?
I know we can specified register as the parameter manually but how about stack?

An example is like in Ghidra where we can specified the stack as the parameter by changing the parameter storage like in the picture below.
![image](https://github.com/Vector35/binaryninja-api/assets/121478081/c29b90fa-7911-408f-b97b-be69e9124527)

additionally, can anyone explain briefly about these, please?
![image](https://github.com/Vector35/binaryninja-api/assets/121478081/755e36d1-6d5b-4a3b-9250-df5510d8f4e6)

Thank you guys

**Comment by plafosse** [comment, marked as answer]
Unfortunately as of now you can not specify this. Follow this issue if you want updates: https://github.com/Vector35/binaryninja-api/issues/3340 Please re-open if I'm mistaken on what you're asking for.

For questions about the edit functions dialog you can look at our documentation here: https://docs.binary.ninja/guide/index.html#edit-function-properties-dialog
